server/endpoints/workspaces.js

```diff
@@ -328,6 +329,58 @@ function workspaceEndpoints(app) {
     }
   );
 
+  app.post(
+    "/workspace/reset-all-vector-dbs",
+    [validatedRequest, flexUserRoleValid([ROLES.admin, ROLES.manager])],
+    async (request, response) => {
+      try {
+        const user = await userFromSession(request, response);
+        const VectorDb = getVectorDbClass();
+
+        const workspaces = multiUserMode(response)
```

Comment:
We can make this some generic Util function instead of an endpoint. Even then, managers should not be able to call this since only admins can modify providers